jena-arq/src/main/java/org/apache/jena/riot/JsonLDReadContext.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
```

Comment:
Please remove the <p> and indent the URL so that the text is exactly the same as other files.

Because this is a new file, it is better that the contributor does this and not the project. License/copyright notices are sacrosanct!